gulpfile.js

```diff
@@ -30,6 +30,12 @@ gulp.task('copy-vendors', function () {
 		.pipe(gulp.dest('./public/js'));
 });
 
+gulp.task('copy-assets', function () {
```

Comment:
Do we expect that repo to get larger exponentially ? 
Was just thinking about philosophy of starting from bare minimum required to accomplish the task?
No crazy future proofing